src/api/app/controllers/webui/status_messages_controller.rb

```diff
@@ -32,7 +44,7 @@ def update
       flash[:error] = "Could not update status message: #{status_message.errors.full_messages.to_sentence}"
     end
 
-    redirect_to(controller: 'main', action: 'index')
+    redirect_to({ action: 'index' })
```

Comment:
Whenever a record is updated, we redirect to the `Manage News` page. Using `redirect_back_or_to` wouldn't work here, so we had to take a decision.